### PR TITLE
About page update

### PR DESCRIPTION
**Pull request by ahujatejas06**
about page updated

don't merge PR as of now

### CODE REVIEW

**Comment by ahujatejas06** [comment]
-closes #94 
-closes #95

File: views/HomePage/Testimonials.tsx [hunk 1]
```diff
@@ -59,7 +59,7 @@ export default function Testimonials() {
 								<Content>{singleTestimonial.content}</Content>
 								<AuthorContainer>
 									<AuthorImageContainer>
-										<NextImage src={singleTestimonial.author.avatarUrl} alt={singleTestimonial.author.name} width={48} height={48} />
+										<NextImage src={singleTestimonial.author.avatarUrl} alt={singleTestimonial.author.name} width={200} height={100} className="imgTestimonial"/>
```
[the remark this answers is not in the document]
Comment (ahujatejas06) [reply views/HomePage/Testimonials.tsx:62]: yes testimonials was reverted as changing the CSS of the styled components was making no change on the output

Will put another PR for issue #92

**Comment by 1-Harshit** [comment]
fix links on footer and student page from `/blog` to `/insights`

**Comment by 1-Harshit** [comment]
fix link on `/insight/index` from `/blog/slug` to `/insights/slug`

File: pages/insights/index.tsx [hunk 2]
```diff
@@ -37,7 +37,7 @@
 					{filteredBlogPosts.map((singlePost, idx) => {
 						if (singlePost.slug.includes("2022-intern")) {
 							return (
-								<NextLink href={"/blog/" + singlePost.slug} passHref key={idx}>
+								<NextLink href={"/insights/" + singlePost.slug} passHref key={idx}>
```
[the remark this answers is not in the document]
Comment (abhishekshree) [reply pages/insights/index.tsx:40]: lol @1-Harshit dekh

**Comment by abhishekshree** [comment]
Check for the security issue relevance.

**Pull request merged**